### PR TITLE
Introduce common rubocop configuration file

### PR DESCRIPTION
**Pull request by jrdufresne**
The common rubocop configuration file will be based on literacyfootprints project and common rubocop extensions across projects.

The common extensions across literacyfootprints, digitalreader-content and preview-manager are:

- rubocop-factory_bot
- rubocop-performance
- rubocop-rspec

All these extensions are dependencies for the `pvb-rubocop` gem. Since these are now dependencies, projects can remove these gems from their `Gemfile`.

To utilize the common configuration add the `pvb-rubocop` gem to the desired projects `Gemfile`. Add the following to the `.rubocop.yml` file:

```yaml
inherit_gem:
  pvb-rubocop: .rubocop.yml
```

Once added remove any duplicated configuration.

### CODE REVIEW

**Comment by jrdufresne** [comment]
Follow up work to utilize the Gem

- https://github.com/Pioneer-Valley-Books/digitalreader-content/pull/947
- https://github.com/Pioneer-Valley-Books/preview-manager/pull/768
- https://github.com/Pioneer-Valley-Books/literacyfootprints/pull/11707

**Review by giverm — DISMISSED** [review]

File: .rubocop.yml
Comment (jdufresne) [inline .rubocop.yml:1]: As this isn't used as a default configuration, I don't see any value in making this a hidden file:

```
git mv .rubocop.yml rubocop.yml
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 1.0.0 (2024-10-08)
```
Comment (jdufresne) [inline CHANGELOG.md:1]:
```suggestion
## 1.0.0 (2024-10-09)
```

File: pvb-rubocop.gemspec [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = 'pvb-rubocop'
+  spec.version = '1.0.0'
+  spec.licenses = ['MIT']
+  spec.summary = 'Pioneer Valley Books Rubocop Configuration'
+  spec.authors = ['Pioneer Valley Books']
+  spec.files = ['.rubocop.yml']
```
Comment (jdufresne) [inline pvb-rubocop.gemspec:9]: Then:

```suggestion
  spec.files = ['rubocop.yml']
```

**Review by jdufresne — DISMISSED** [review]
Nice! Thanks.

**Comment by jrdufresne** [comment]
@giverm @jdufresne addressed freedback in latest

**Review by giverm — APPROVED** [review]

**Review by jdufresne — APPROVED** [review]

**Pull request merged**